web/src/features/panels/zone/ZoneHeaderTitle.tsx

```diff
@@ -14,12 +14,18 @@ interface ZoneHeaderTitleProps {
   isAggregated?: boolean;
 }
 
+const MAX_TITLE_LENGTH = 25;
+
 export default function ZoneHeaderTitle({ zoneId }: ZoneHeaderTitleProps) {
-  const title = getZoneName(zoneId);
+  const zoneNameShort = getZoneName(zoneId);
```

Comment:
There is no guarantee this is actually shorter since it only would be if there is a display name property for the zone.

I think we should just rename it to zoneDisplayName (and the other to zoneFullName).